Allow standalone coverlet usage for integration/end-to-end tests using .NET tool driver (#991)
coverlet-coverage/coverlet: README.md

* VSTest engine integration
* MSBuild task integration

> * As a .NET Global tool(used also for integration test, end-to-end tests)

Reviewer: ```suggestion
* As a .NET Global tool (supports standalone integration tests)
```
